### PR TITLE
Added latest projects

### PR DESCRIPTION
**Pull request by atticoos**
This adds [Field the Bern](https://fieldthebern.com) and [Bernie BNB](http://www.berniebnb.com) projects.

screen shot 2016-01-05 at 9 22 03 pm

### CODE REVIEW

File: projects/field-the-bern.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: page
+title: Field the Bern
+permalink: /projects/field-the-bern/
+---
+
+<img src="/img/field-the-bern.png" alt="Field the Bern" />
+
+[Field the Bern](https://fieldthebern.com/) is a canvassing mobile application to allow anyone to dive right into local canvassing. Users can log in and immediately browse the area to see what houses have been visited and which haven't. After knocking on a door and speaking with a resident, canvassers can record their interaction and gain points. Track your progress and compete with friends. Helping Bernie win is even more fun with this app!
```
Comment (atticoos) [inline projects/field-the-bern.md:9]: /cc @JoshSmith on this summary

File: projects/bernie-bnb.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+layout: page
+title: Bernie BNB
+permalink: /projects/bernie-bnb/
+---
+
+<img src="/img/bernie-bnb.png" alt="Bernie BNB" />
+
+[Bernie BNB](http://www.berniebnb.com) is a web application to allow volunteers to share their homes for those traveling in support for Bernie Sanders.
```
Comment (atticoos) [inline projects/bernie-bnb.md:9]: /cc @SandersForPresident/berniebnb-maintainers on this summary. I'm sure there can be more to add

**Comment by schneidmaster** [comment]
LGTM. Merging for now, if people tagged above have comments, feel free to submit a separate PR.

**Comment by atticoos** [comment]
:+1: ty

**Pull request merged**